geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/iterators/MultiIteratorTest.scala

```diff
@@ -33,6 +35,10 @@ class MultiIteratorTest extends Specification with TestWithMultipleSfts with Laz
 
   val spec = SimpleFeatureTypes.encodeType(TestData.featureType, includeUserData = true)
 
+  // noinspection LanguageFeature
+  // note: size returns an estimated amount, instead we need to actually count the features
+  implicit def collectionToIter(c: SimpleFeatureCollection): Iterator[SimpleFeature] = SelfClosingIterator(c)
```

Comment:
Let's try SelfClosingIterator.  I've had some requests to be able to close resources and having access to the proper 'close' method seems beneficial.